### ISSUE TITLE
missing db fields and www prefix

### ISSUE THREAD

**Issue by musicpanda** [thirtybees/coreupdater#23]
I updated my Prestools website to the latest version. Several things went wrong.

The first thing I noticed was that while updating the process was not finished. It ended with me getting a message "Ajax request failed: error". Unfortunately the place where the error should should was empty. When I refreshed the page it said that my site was up-to-date. However, when I tried to create a customer account it crashed during the saving. I tried bleeding edge but that didn't help.

Changing a file allowed me to "upgrade" again. Now I had the javascript console open and I saw that there was a CORS error. I was running on domain "prestools.com" and from there a call was made for a file from "www.prestools.com" and that was refused by the browser for cross scripting reasons.

Going back to the front side I ran in database problems. First the field cl.display_name was missing and then category_shop.active. After that the shop seemed to run. But unfortunately there is no way to check whether there are more database differences. The updater script doesn't check for it and unlike Prestashop Thirty Bees doesn't document the database version differences in an accessible way.

**Comment by musicpanda** [comment]
I forgot the image of the error message
![coreupd](https://user-images.githubusercontent.com/13930107/168605108-72adbfa9-cdaa-4625-b804-6759d418a02d.png)

**Comment by musicpanda** [comment]
This is how it looks after the failed upgrade.
![coreupd](https://user-images.githubusercontent.com/13930107/168994271-71d884f7-c46e-46b9-8d11-df8c35a1cf4d.jpg)

**Comment by getdatakick** [comment]
This is duplicate of issue #17. 

The store update was actually performed in CORS preflight HEAD request (this is unintentional, should be fixed). The following POST request was blocked by CORS mechanism, and this in turn blocked all subsequent operations, including db update.

Your reported issues will be fixed as part of #17 and #18
